datafusion-cli/Cargo.toml

```diff
@@ -25,7 +25,7 @@ keywords = ["arrow", "datafusion", "query", "sql"]
 license = "Apache-2.0"
 homepage = "https://github.com/apache/arrow-datafusion"
 repository = "https://github.com/apache/arrow-datafusion"
-rust-version = "1.70"
+rust-version = "1.73"
```

Comment:
It is fairly standard practice to set the MSRV to accommodate at least a couple of older versions, I'm not sure what DataFusion's policy is, but updating to 1.73 which was only released 9 days ago would be a very aggressive posture.

For comparison arrow-rs has an MSRV of 1.62

Reply:
agree